From f3f75d01d96c1224ffe6168821d3486a50bc05db Mon Sep 17 00:00:00 2001
From: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
Date: Mon, 28 Sep 2020 20:39:45 +0530
Subject: [PATCH] Fix signapk.jar path for RELEASE_BUILD=true case

This patch fixes the issue with signapk.jar path.
This path was set to the script directory but needed to be
set to the output directory.
Without this if code built with RELEASE_BUILD=true will fail.
As the cass-sign****.zip will not be generated and signapk.jar
will not be found.

Change-Id: I40cfea80f3b23a8e73c7bbb27ba3fb2bc47a3158
Tracked-On: OAM-92994
Signed-off-by: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
---
 tools/releasetools/common.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/releasetools/common.py b/tools/releasetools/common.py
index 897c4bdf7d..3b840a635b 100644
--- a/tools/releasetools/common.py
+++ b/tools/releasetools/common.py
@@ -64,7 +64,7 @@ class Options(object):
           'executable -- build and run `{}` directly.'.format(
               script_name[:-3]),
           file=sys.stderr)
-    self.search_path = os.path.dirname(os.path.dirname(exec_path))
+    self.search_path = os.path.realpath(os.path.join(os.path.dirname(exec_path), '../../../../out/host/linux-x86'))
 
     self.signapk_path = "framework/signapk.jar"  # Relative to search_path
     self.signapk_shared_library_path = "lib64"   # Relative to search_path
-- 
2.17.1

